src/main/java/org/candlepin/subscriptions/security/IdentityHeaderAuthenticationDetailsSource.java

```diff
@@ -83,12 +91,12 @@ public IdentityHeaderAuthenticationDetailsSource(ObjectMapper objectMapper,
             Map identity = (Map) authObject.getOrDefault("identity", Collections.emptyMap());
             Map user = (Map) identity.getOrDefault("user", Collections.emptyMap());
 
-            if (((Boolean) user.getOrDefault("is_org_admin", Boolean.FALSE))) {
-                userRolesList.add(ORG_ADMIN);
+            if (((Boolean) user.getOrDefault("is_org_admin", Boolean.FALSE)) || props.isDevMode()) {
```

Comment:
Sonarqube failing on this and the following similar if statement...

After rewriting, looks like 
```java
if (Boolean.TRUE.equals(user.get("is_org_admin")) || props.isDevMode()) {
``` 
will work (and I find that a little easier to read personally).

Reply:
:+1:

Reply:
done